src/en/juju-upgrade.md

```diff
@@ -1,197 +0,0 @@
-
-When not specifying a version to upgrade to (`--version`) an algorithm will be
-used to auto-select a version.
-
-Rules:
-
-1. If the server major version matches the client major version, the version
-   selected is minor+1. If such a minor version is not available then the next
-   patch version is chosen.
-
-1. If the server major version does not match the client major version, the
-   version selected is that of the client version.
-
-To demonstrate, let the available online versions be: 1.23, 1.24, 1.25, 1.25.1,
-2.1, and 2.2. This gives:
-
-- client 1.24, server 1.22 -> upgrade to 1.23
-- client 2.1, server 1.22 -> upgrade to 2.1
-- client 1.24, server 1.25 -> upgrade to 1.25.1
-
-The online server software is found here:
-
-- stable: https://streams.canonical.com/juju/tools/releases/
-- devel: https://streams.canonical.com/juju/tools/devel/
-
-### The sync-tools command
-
-This pertains to the lack of internet connectivity for the state server.
-
-If the state server cannot contact the above URLs then the client, which
-presumably does have access, will need to download and transfer the software
-to the state server prior to requesting an upgrade. This is accomplished with
-the `sync-tools` command.
-
-Examples:
-
-Transfer the server software (auto-selected) to the state server:
-
-```bash
-juju sync-tools
-```
-
-Transfer the specified server software (all patch versions of 1.23) to the
-state server:
-
-```bash
-juju sync-tools --version 1.23 --debug
-```
-
-!!! Note: the `sync-tools --version` command only accepts `major[.minor]`
-("e.g. use '1.24' not '1.24.5').
-
-For complete syntax, see the [command reference page](./commands.html#sync-tools)
-or by running `juju help sync-tools`.
-
-### The upgrade-juju command
-
-The `upgrade-juju` command initiates the upgrade of the server software. This
-will cause all Juju agents to be restarted. Before proceeding, ensure the
-environment is in good working order (`juju status`).
-
-Examples:
-
-Upgrade the environment by allowing the version to be auto-selected:
-
-```bash
-juju upgrade-juju --debug
-```
-
-Upgrade the environment by specifying the version:
-
-```bash
-juju upgrade-juju --version 1.25.1 --debug
-```
-
-Track the progress with
-
-```bash
-watch -n3 "juju status --format tabular"
-```
-
-For complete syntax, see the [command reference page](./commands.html#upgrade-juju)
-or by running `juju help upgrade-juju`.
-
-!!! Warning: The `--upload-tools` option should be not be used by the end user.
-
-
-## Verifying the upgrade
-
-The verification of a successful upgrade is obtained by:
-
-```bash
-juju status
-```
-
-If this command does not complete properly or if there are errors displayed in
-its output then proceed to the next section.
-
-
-## Troubleshooting the upgrade
-
-An upgrade of server software that does not lead to 100% success will require
-troubleshooting. See [Troubleshooting environment upgrades](./troubleshooting-upgrade.html).
```

Comment:
copy the changed one somewhere else if you need to keep it, and copy a version of the unchanged one back in.